game-core/src/main/java/games/strategy/engine/data/GameData.java

```diff
@@ -104,7 +102,6 @@
 
   public GameData() {
     delegateList = new DelegateList(this);
-    properties.set(GAME_UUID, UUID.randomUUID().toString());
```

Comment:
Since ladders and tournaments have been talked about recently, I was just wondering if the UUID is a valuable attribute of `GameData` in those cases?

I have no objection to removing this specific implementation of storing a unique game ID.  My guess is Sean added it to `properties` in 5ab9c2801 for compatibility reasons instead of simply adding a new field.  If we need it for tourneys/ladders in the future, I would suggest we add it back as a named field rather than a game property.